src/components/ProblemAdministration/ProblemAdministration.tsx

```diff
@@ -112,28 +113,30 @@ export const ProblemAdministration: FC = () => {
                   />
                 </td>
                 <td>
-                  {solution.solution ? (
-                    <a href={solution?.solution} target="_blank" rel="noreferrer">
-                      <FormatAlignJustify />
-                    </a>
-                  ) : (
+                  <div className={styles.buttonsCell}>
```

Comment:
hm, mozno by tie styly sli hodit aj priamo na `td`, ze by netrebalo dalsi `div`, dunno

Reply:
Keď som zmenit display na flex tak sa `td` fungovalo veľmi divne.

Reply:
vsak som si vravel, ze... mozno 😄 a mozno bude problem 😄

Reply:
kazdopadne prerobim to asi cele na grid v dalsom PR, takze verim ze odsranim tie vecci navyse